rust/src/lib/nispor/route.rs

```diff
@@ -209,6 +209,7 @@ fn np_route_to_nmstate(np_route: &nispor::Route) -> RouteEntry {
     route_entry.initcwnd = np_route.initcwnd;
     route_entry.initrwnd = np_route.initrwnd;
     route_entry.mtu = np_route.mtu;
+    route_entry.quickack = np_route.quickack.map(|q| q > 0);
```

Comment:
If the quickack only meaningful when the quickack is true? Why do we always omit quickack false when showing the property?

Reply:
default is `quickack: false`.